languagetool-core/src/main/java/org/languagetool/Languages.java

```diff
@@ -283,7 +285,7 @@ public static Language getLanguageForLocale(Locale locale) {
         return firstFallbackLanguage;
       }
     }
-    for (Language aLanguage : languages) {
```

Comment:
Why `getLanguageForLocale(Locale locale) ` exclude dynamic language module?

Reply:
LT assumes en-US is static language module. Now the change allow dynamic loading of even en-US for fallback.